app/lib/profile/pages/profile.dart

```diff
@@ -1,12 +1,31 @@
-import 'package:flutter/material.dart';
+import 'package:flutter/material.dart' hide MetaData;
+
+import '../../common/models/metadata.dart';
+import '../../common/models/userdata.dart';
 
 class ProfilePage extends StatelessWidget {
   const ProfilePage({Key? key}) : super(key: key);
 
+// ignore: flutter_style_todos
+// TODO: remind me to remove this, for debugging only!
+  String debug() {
+    // ignore: prefer_single_quotes
+    return """
+    ${UserData.instance.diplotypes?[0].gene}
+    ${UserData.instance.diplotypes?.length}
+
+    ${UserData.instance.lookups?.length}
+    ${UserData.instance.lookups?[0].keys.first} : ${UserData.instance.lookups?[0].values.first}
+
+    ${MetaData.instance.isLoggedIn}
+    ${MetaData.instance.lookupsLastFetchDate}
+    """;
```

Comment:
```suggestion
    return '''
    ${UserData.instance.diplotypes?[0].gene}
    ${UserData.instance.diplotypes?.length}

    ${UserData.instance.lookups?.length}
    ${UserData.instance.lookups?[0].keys.first} : ${UserData.instance.lookups?[0].values.first}

    ${MetaData.instance.isLoggedIn}
    ${MetaData.instance.lookupsLastFetchDate}
    ''';
```

Remove this or change to single quotes